src/pages/Todo.tsx

```diff
@@ -1,7 +1,54 @@
-import React from 'react'
+import { useState } from 'react';
 
-const Todo = () => {
-  return <div>todo</div>
+export interface Todo {
+  id: number;
+  todo: string;
+  isCompleted: boolean;
+  userId: number;
 }
 
-export default Todo
+const DUMMY_TODOS: Todo[] = [
+  {
+    id: 1,
+    todo: '투두 1',
+    isCompleted: false,
+    userId: 1,
+  },
+  {
+    id: 2,
+    todo: '투두 2',
+    isCompleted: true,
+    userId: 1,
+  },
+  {
+    id: 3,
+    todo: '투두 3',
+    isCompleted: false,
+    userId: 1,
+  },
+];
+
+const Todo = () => {
+  const [todoData, setTodoData] = useState({
+    loading: false,
+    todos: DUMMY_TODOS,
+    error: null,
+  });
```

Comment:
타입을 하나 추가하는게 어떨까요? 
```suggestion
  interface TodoDataType {
    loading: boolean,
    todos: Todo[],
    error: null | Error
  }
  const [todoData, setTodoData] = useState<TodoDataType>({
    loading: false,
    todos: DUMMY_TODOS,
    error: null,
  });
```

근데 로딩이랑 에러 상태를 따로 만들어서 저렇게 처리하신 이유가 있나요?
로딩이랑 에러는 언제 쓰여요?

Reply:
타입 추가하겠습니다! 로딩과 에러는 비동기로 투두데이터를 받아올 때와 받아오지 못했을 때의 상태를 관리하기 위해 작성했습니다.

Reply:
API로 데이터를 받아올 때 그 지연시간이 조금 있는데 그 지연시간 또한 표시할 로딩 컴포넌트와 API로 데이터를 받아오는데 실패했을 때 표시할 에러 컴포넌트를 렌더링하기 위해 따로 상태를 만들었습니다.

Reply:
일단 저희의 의견은 이런데 혹시 어떻게 생각하시나요?